File: content/learn/arrays-easy/04-remove-duplicates/starter.js
function removeDuplicates(a) {
  // TODO: dedupe a in place and return the new length.
  return 0;
}

const n = parseInt(readline(), 10);
const a = n > 0 ? readline().split(" ").map(Number) : [];
const k = removeDuplicates(a);
print(k);
print(a.slice(0, k).join(" "));

File: content/learn/arrays-easy/04-remove-duplicates/starter.py
def remove_duplicates(a):
    # TODO: dedupe a in place and return the new length.
    return 0


n = int(input())
a = list(map(int, input().split())) if n > 0 else []
k = remove_duplicates(a)
print(k)
print(" ".join(str(x) for x in a[:k]))
